faq/tutum-on-aws-faq.md

```diff
@@ -4,22 +4,40 @@ The following objects will be created automatically by Tutum when deploying a no
 
 * An SSH keypair named `tutum-<uuid>`
 * An EBS volume of type `gp2` with the size specified on the **Create new node cluster** wizard
-* A security group named `tutum-default` with all ports open by default
 
-By default, Tutum will try to deploy instances on EC2 using "EC2-Classic" or "EC2-VPC" using the default VPC for your account in the selected region. If there's no default VPC, Tutum will create the following additional objects:
+By default, Tutum will try to deploy instances on EC2 using "EC2-VPC" ("EC2-Classic" is deprecated). It will create the following additional objects:
 
-* A VPC named `tutum-vpc` with the CIDR range `172.77.0.0/16`
 * An internet gateway named `tutum-gateway` attached to `tutum-vpc`
-* A subnet named `tutum-subnet` with the CIDR range `172.77.0.0/16` in `tutum-vpc`
 * A route table named `tutum-route-table` in `tutum-vpc` associated with `tutum-gateway` and `tutum-subnet`
+
+There are three ways to create the rest of the objects:
+
+### Leverage Tutum in order to create them automatically
+
+Tutum will create them by default according to the following rules:
+
+* A VPC is created with the tag name `tutum-vpc` and with the CIDR range `10.77.0.0/16`
+* A set of subnets, if there are no subnets already created in the VPC. Tutum will create a subnet in every AZ possible, and will leave enough CIDR space for the user to create customized subnets. Every subnet created is tagged with the name `tutum-subnet`
 * A VPC security group named `tutum-vpc-default` with all ports open by default
 
 In order for Tutum to detect these objects after they are created, it's important that **their names don't change**. Otherwise, Tutum will try to recreate them and fail due to conflicts.
 
+### Select them through UI or API
+
+This gives you full control of the resources that Tutum will use to deploy your nodes, choosing the VPC, the subnet/s and the security group. You can do it through Tutum's user interface or [API](https://docs.tutum.co/v2/api/#create-a-new-node-cluster)
+
+### Have them already created
```

Comment:
We should discourage this as the user can select it manually now. Remove this section.